module/files/helper.js

```diff
@@ -7,404 +7,444 @@
  */
 
 // Dependencies
-const Zip = require("adm-zip");
-const fs = require("fs");
-const awsServices = require(ROOT_PATH + "/generics/services/aws");
-const googleCloudServices = require(ROOT_PATH + "/generics/services/google-cloud");
-const azureService = require(ROOT_PATH + "/generics/services/azure");
+const Zip = require('adm-zip')
```

Comment:
@rakeshSgr - Add semi-colons at the end of line

Reply:
resolved